chipsec/hal/uefi_platform.py

```diff
@@ -796,7 +796,7 @@ def EFIvar_EVSA(nvram_buf):
     sn = 0
     EVSA_RECORD = "<IIII"
     evsa_rec_size = struct.calcsize(EVSA_RECORD)
-    GUID_RECORD = "<HIHH8s"
+    GUID_RECORD = "<H4s2s2s8s"
```

Comment:
Why <H4s2s2s8s and not <H4s2s2s2s6s? Later on you're just dicing the last variable up anyways:
def guid_str(guid0, guid1, guid2, guid3):
    guid = uuid.UUID(bytes=guid0[::-1] + guid1[::-1] + guid2[::-1] + guid3[:2] + guid3[-6:])

Reply:
I was initially trying to do minimal touching of the old code.   Your change makes sense, but I would actually like all guid to use uuid.UUID.